### ISSUE TITLE
GUI for the tool (optional)

### ISSUE THREAD

**Issue by lukaswolfrum** [amosproj/amos2022ss07-nft-playbook#23]
## User story
1. As a power user
2. I want / need to navigate through the tool in an understandable manner
3. So that non-tech people can follow the process

## Acceptance criteria
* All functionalities of the CLI are doable over the GUI
* The GUI looks appealing 
* The GUI is navigateable and understandable for non-tech people
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
